### DISCUSSION TITLE
Wrong audio in video on multi language audio videos

### DISCUSSION THREAD

**Discussion by Fuzzlix** [aclap-dev/video-downloadhelper#1299] [Help and Q&A] [answered]
### Your question

This youtube video has AI generated translated audio tracks:
[video](https://www.youtube.com/watch?v=0GgqQBoe5ts&list=PLpTTF6wMDLR6mt7g_yE7PVSkSVreGLipr)
Watching this video im Firefox, i hear my language (german) audio. of course i can choose different audio languages if i want to.
If i try to download the video, i allways download the video with arabic audio, no matter what langage i am listen in Firefox.

### Addon & CoApp version and details

version: 9.1.0.41
target: mozilla
channel: beta
lang: de
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"a1ee1d0bb13a4afd"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "valid_license_message_has_been_displayed": true,
    "http_media_download_strategy": "coapp",
    "used_history_button": true,
    "download_directory": "C:\\Users\\uschz\\Videos",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "S:\\MediathekView"
        }
    },
    "concurrent_downloads_max": 2,
    "show_thumbnail_in_notification": false,
    "show_success_notification": false,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": true
    },
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 10,
    "successfull_dl": 60,
    "never_show_successfull_dl_message": true,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_6047618364110966"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Star Trek - Enterprise - Countdown.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Star Trek - Enterprise - Countdown.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/star-trek-enterprise/countdown"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1726771663772
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_5476967843136143"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Star Trek - Das nachste Jahrhundert - Der zeitreisende Historike.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Star Trek - Das nachste Jahrhundert - Der zeitreisende Historike.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/star-trek-das-naechste-jahrhundert/der-zeitreisende-historiker"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1726773817813
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_8001438118642905"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Raumschiff Enterprise - Das Jahr des roten Vogels.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Raumschiff Enterprise - Das Jahr des roten Vogels.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/raumschiff-enterprise/das-jahr-des-roten-vogels"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727154838076
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_3069878050315407"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Raumschiff Enterprise - Stein und Staub.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Raumschiff Enterprise - Stein und Staub.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/raumschiff-enterprise/stein-und-staub"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727157920626
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_8982091066057584"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Star Trek - Enterprise - Sturmfront - Teil 1.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Star Trek - Enterprise - Sturmfront - Teil 1.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/star-trek-enterprise/sturmfront-teil-1"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727159433412
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_396334946641707"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Star Trek - Das nachste Jahrhundert - Der einzige Uberlebende.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Star Trek - Das nachste Jahrhundert - Der einzige Uberlebende.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/star-trek-das-naechste-jahrhundert/der-einzige-uberlebende"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727161852240
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_6135107979654922"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Star Trek - Enterprise - Sturmfront - Teil 2.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Star Trek - Enterprise - Sturmfront - Teil 2.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/star-trek-enterprise/sturmfront-teil-2"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727219518733
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_5906829822484137"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Star Trek - Das nachste Jahrhundert - Geistige Gewalt.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Star Trek - Das nachste Jahrhundert - Geistige Gewalt.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/star-trek-das-naechste-jahrhundert/geistige-gewalt"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727221460596
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_3537079351465073"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Star Trek - Enterprise - Zuhause.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Star Trek - Enterprise - Zuhause.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/star-trek-enterprise/zuhause"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727292167912
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_4982846064490257"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Star Trek - Das nachste Jahrhundert - Das kunstliche Paradies.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Star Trek - Das nachste Jahrhundert - Das kunstliche Paradies.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://tele5.de/mediathek/star-trek-das-naechste-jahrhundert/das-kunstliche-paradies"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727293066535
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_2477419690302233"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\HAMMERHART Gerhard Schroder sagt das UNSAGBARE - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "HAMMERHART Gerhard Schroder sagt das UNSAGBARE - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=1u9-NHa5Aw0"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727301602883
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_3068322857071875"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\ZEIT-Matinee mit Gerhard Schroder vom 09 03 2014 - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "ZEIT-Matinee mit Gerhard Schroder vom 09 03 2014 - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=EKQ0ykFQav4&t=4s"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727301894779
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_7452156913905226"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\KI im Krieg Was in Israel und der Ukraine passiert I auslandsjournal frontlines - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "KI im Krieg Was in Israel und der Ukraine passiert I auslandsjournal frontlines - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=wZaqsgbqAm8"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727306492447
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_280711803585360"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Meine Lieblingsrezepte - so richtig guter Kartoffelsalat mit Frikadellen - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Meine Lieblingsrezepte - so richtig guter Kartoffelsalat mit Frikadellen - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=JqO38dnNWDc&list=PL4TYpjhKw5RJaqkgV3w5Wb3d-EDj-F-Kb&index=3"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727310029455
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_3675129152933416"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Toastbrot bzw Buttertoast mit Roggensauerteig - ein Toastbrot kann auch lecker UND vollwertig sein - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Toastbrot bzw Buttertoast mit Roggensauerteig - ein Toastbrot kann auch lecker UND vollwertig sein - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=SOrLNn-xZFY&list=PL4TYpjhKw5RJaqkgV3w5Wb3d-EDj-F-Kb&index=11"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727312671978
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_3489467595353644"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Lose Belly Fat But Dont Do These Common Exercises 5 Minute 10 Day Challenge - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Lose Belly Fat But Dont Do These Common Exercises 5 Minute 10 Day Challenge - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=0GgqQBoe5ts"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727328658571
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_8462464770815414"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Blumenkohl Walnuss Muffins - spannende Kombination von Zutaten fur ein phant Geschmackserlebnis - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Blumenkohl Walnuss Muffins - spannende Kombination von Zutaten fur ein phant Geschmackserlebnis - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=JqO38dnNWDc&list=PL4TYpjhKw5RJaqkgV3w5Wb3d-EDj-F-Kb&index=5"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727331465231
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_315047505043812"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Sauerteig Profiwissen - mit Sauerteig abnehmen MIt Rezepten fur Mohren Brot und Pastinaken Kuchen - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Sauerteig Profiwissen - mit Sauerteig abnehmen MIt Rezepten fur Mohren Brot und Pastinaken Kuchen - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=SOrLNn-xZFY&list=PL4TYpjhKw5RJaqkgV3w5Wb3d-EDj-F-Kb&index=12"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727332067017
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_8005854362148715"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\10 HIIT-Ubungen zur Fettverbrennung - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "10 HIIT-Ubungen zur Fettverbrennung - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=1Qongso4FvM&list=PLpTTF6wMDLR6Be5d98HRh0Y1xmrCDq9zy&index=7"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727336025093
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_930426446744707"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\What is HIIT Training Why HIIT Works Fast Weight Loss With High Intensity Interval Training - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "What is HIIT Training Why HIIT Works Fast Weight Loss With High Intensity Interval Training - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=EOlgg1Ud-UM&list=PLpTTF6wMDLR6Be5d98HRh0Y1xmrCDq9zy&index=4"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727336419928
                        }
                    }
                ]
            },
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "downloadable_4844439262404478"
                    },
                    {
                        "download_result": {
                            "inbrowser": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            },
                            "filepath": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos\\Lose Belly Fat But Dont Do These Common Exercises 5 Minute 10 Day Challenge - YouTube.mp4"
                            },
                            "filename": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "Lose Belly Fat But Dont Do These Common Exercises 5 Minute 10 Day Challenge - YouTube.mp4"
                            },
                            "filedir": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": "C:\\Users\\uschz\\Videos"
                            },
                            "qrcode": {
                                "__serializer_tag": "primitive",
                                "__serializer_value": false
                            }
                        },
                        "page_url": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": "https://www.youtube.com/watch?v=0GgqQBoe5ts&list=PLpTTF6wMDLR6mt7g_yE7PVSkSVreGLipr"
                        },
                        "timestamp": {
                            "__serializer_tag": "primitive",
                            "__serializer_value": 1727336607720
                        }
                    }
                ]
            }
        ]
    },
    "record_download_history": true
}

**Comment by Mattmjs** [comment, marked as answer]
Probably Arabic is the first audio track for each resolution therefore that's what the software picks when you download.
I don't know what could be done on these videos other than to use something like **yt-dlp**.
You can bring up all video & audio formats and choose which ones you want.

**Reply by Fuzzlix** [reply]
yt-dlp seems to solve my problem. Thank you for your help.
